templates/android.js

```diff
@@ -9,17 +12,13 @@ module.exports = platform => [{
     dependencies {
         // Matches recent template from React Native (0.59)
         // https://github.com/facebook/react-native/blob/0.59-stable/template/android/build.gradle#L16
```

Comment:
```suggestion
        // https://github.com/facebook/react-native/blob/0.60-stable/template/android/build.gradle#L16
```

(please double-check that my change to the link is correct)